hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyOutputStream.java

```diff
@@ -440,6 +441,19 @@ public void flush() throws IOException {
     handleFlushOrClose(StreamAction.FLUSH);
   }
 
+  @Override
+  public void hflush() throws IOException {
+    hsync();
+  }
+
+  @Override
+  public void hsync() throws IOException {
+    checkNotClosed();
+    handleFlushOrClose(StreamAction.HSYNC);
+    //TODO send hsyncKey to update length;
+    //     see blockOutputStreamEntryPool.commitKey(offset);
```

Comment:
I think, later we can add a blockOutputStreamEntryPool.hsync(), which is mostly the same as commitKey except that it does not support multi part upload.